common/djangoapps/student/models.py

```diff
@@ -1621,6 +1668,12 @@ def user_can_skip_entrance_exam(cls, user, course_key):
 class LanguageProficiency(models.Model):
     """
     Represents a user's language proficiency.
+
+    Note that we have not found a way to emit analytics change events by using signals directly on this
+    model or on UserProfile. Therefore if you are changing LanguageProficiency values, it is important
```

Comment:
This is a little confusing, since you have registered a post_save receiver above for UserProfile... is this comment just out of date?